src/components/AddFacilityModal/AddFacilityModal.jsx

```diff
@@ -30,6 +33,41 @@
 const AddFacilityModal = () => {
   const { isOpen, onOpen, onClose } = useDisclosure();
   const [scrollBehavior] = React.useState('inside');
+  const [modalIsOpen, setModalIsOpen] = useState(false);
```

Comment:
🚫 **[eslint]** <[no-undef](https://eslint.org/docs/rules/no-undef)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'useState' is not defined.